### ISSUE TITLE
Are the data from these fields always increasing or decreasing?

### ISSUE THREAD

**Issue by andronsay** [mathdroid/covid-19-api#128]
Are the data from these fields (example, https://covid19.mathdro.id/api/countries/usa -> confirmed, recovered) always increasing or decreasing? These data (deaths), I am sure that they are not decreasing.

**Comment by vitorboccio** [comment]
I opened a similar issue requesting understanding how these numbers works. The short answer to your questions is: no.
In some cases it can be a negative number. Recovered: '-10', for example. What I don't quite understand yet is for example if recovered -1 means death + 1.

**Comment by andronsay** [comment]
> I opened a similar issue requesting understanding how these numbers works. The short answer to your questions is: no.
> In some cases it can be a negative number. Recovered: '-10', for example. What I don't quite understand yet is for example if recovered -1 means death + 1.

Ok get it, thanks. It would be great if the data (confirmed, recovered, deaths) were always static (as if a data bank that saves but does not remove, does not subtract), and people themselves, using mathematics, calculated how much let's recover. And one more time I notice that sometimes during the call let's say (https://covid19.mathdro.id/api) the data is gradually calculated and changed until the calculation is completed. For example, during the calculation, there may be 200,000, 300,000, 400,000, and so on until the final count. Have you ever thought of setting some kind of calculation indicator, such as (workon => true) until the calculation is complete? In order for the user to receive data and understand that he did not get at the time of counting and did not receive incomplete data.

**Comment by vitorboccio** [comment]
Yes. This is happening to me as well and some responses seems to be cached. Definitely needs some improvements on the data accuracy.
